<launch>

   
    <node name = "serialnode" pkg = "rosserial_arduino" type = "serial_node.py">
    <param name="port" value="/dev/ttyUSB0"/>
    
    <node name = "output" pkg = "materov22_pioneer" type = "output"/>

    <node name = "motion_command_executer" pkg = "materov22_pioneer" type = "motion_command_executer"/>

    <include ns="camera_node" file="$(find materov22_pioneer)/src/camera/launch/elp.launch" />    
  
  </node>

</launch>